backend/entityservice/views/run/description.py

```diff
@@ -31,6 +32,7 @@ def get(project_id, run_id):
 def _delete_run(run_id, log):
     # Retry if a db integrity error occurs (e.g. when a worker is writing to the same row)
     # Randomly wait up to 2^x * 1 seconds between each retry until the range reaches 60 seconds, then randomly up to 60 seconds afterwards
+    set_run_state_deleted(run_id)
```

Comment:
Are we doing anything different if the run is missing or deleted? If no, could we simply delete the key from redis?

Reply:
A missing run means something has gone wrong where a deletion is normal behavior. 

I could be convinced but I'd prefer to set the state in redis so the backend tasks can act accordingly - perhaps throw a big tantrum if the run is missing, but just exit gracefully if the run has been deleted.